R/map_df.R

```diff
@@ -123,7 +123,7 @@ map_df <- function(trt_md,
 
   out <- lapply(seq_along(trt_rnames), function(i) {
```

Comment:
Proposal - to be consistent: line 116-117
instead of
```
  out <- if (!anyNA(exact_out) && is.null(override_untrt_controls)) {
    exact_out
```
maybe
```
if (!anyNA(exact_out) && is.null(override_untrt_controls)) {
    out <- exact_out
```

Reply:
Lines 95-113 have the same format so I would keep it the same for consistency